when installing a fresh sqlite-database migrations are broken

```
Applying a4_candy_questions.0005_rename_app_in_phase_type...Traceback (most recent call last):
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/apps/registry.py", line 155, in get_app_config
    return self.app_configs[app_label]
KeyError: 'a4phases'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/core/management/commands/migrate.py", line 234, in handle
    fake_initial=fake_initial,
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/db/migrations/executor.py", line 117, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/db/migrations/executor.py", line 245, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/db/migrations/migration.py", line 124, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/home/katharina/a4-speakup/apps/questions/migrations/0005_rename_app_in_phase_type.py", line 7, in rename_app
    Phase = apps.get_model("a4phases", "Phase")
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/apps/registry.py", line 205, in get_model
    app_config = self.get_app_config(app_label)
  File "/home/katharina/a4-speakup/venv/lib/python3.7/site-packages/django/apps/registry.py", line 162, in get_app_config
    raise LookupError(message)
LookupError: No installed app with label 'a4phases'.
make: *** [Makefile:34: install] Error 1
```

--- a/apps/questions/migrations/0005_rename_app_in_phase_type.py
+++ b/apps/questions/migrations/0005_rename_app_in_phase_type.py
@@ -25,6 +25,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('a4_candy_questions', '0004_rename_table'),
+        ('a4phases', '0006_remove_weight_from_phase_type')
     ]
 
     operations = [

